code/lib/builder-vite/src/utils/without-vite-plugins.ts

```diff
@@ -1,16 +1,24 @@
 import type { PluginOption } from 'vite';
 
-// recursively remove all plugins with the given names
-export const withoutVitePlugins = (
+/**
+ * Recursively removes all plugins with the names given
+ * Resolves async plugins
```

Comment:
It doesn't matter that `withoutVitePlugins` will resolve all async plugins, right?  I can't imagine why it would make a difference.

Reply:
I've asked the Vite maintainers and Ben said it didn't make a difference for Svelte at least. I just wanted to point out the behavior explicitly to the next person.